.gitignore

```diff
@@ -0,0 +1 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Well, I like to have .gitignore in order not to accidentially commit .idea folder. How do you guys handle this? Via IDE-Settings?